MyRequestClient.js

```diff
@@ -1,86 +1,73 @@
-'use strict';
+"use strict";
```

Comment:
nit: it might help reduce the # of lines changed to stick with single quotes

Reply:
Good call. I'll update.